From 89a093eb921b437df450cea73b6f12ea5cd0bc67 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 7 Dec 2018 14:36:32 +0900
Subject: [PATCH 571/909] arm64: dts: renesaas: r8a77990: Fixup power-domains
 of 3DGE for R-Car E3 ES1.0

commit f29f638ed3346b4e57bfeaa3153aa3a29370990a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds a workaround for R-Car E3 ES1.0 3DGE power-domains.

Support for ES1.0 revision of R-Car E3 SoC is separated into another
file r8a77990-es10.dtsi. To avoid duplication, r8a77990-es10.dtsi
includes r8a77990.dtsi, and adds device nodes and properties where needed.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-es10.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 arch/arm64/boot/dts/renesas/r8a77990-es10.dtsi

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-es10.dtsi b/arch/arm64/boot/dts/renesas/r8a77990-es10.dtsi
new file mode 100644
index 000000000000..7b067ffdd142
--- /dev/null
+++ b/arch/arm64/boot/dts/renesas/r8a77990-es10.dtsi
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for the R-Car E3 (R8A77990) ES1.0 SoC
+ *
+ * Copyright (C) 2018 Renesas Electronics Corp.
+ */
+
+#include "r8a77990.dtsi"
+
+&gsx {
+	power-domains = <&sysc R8A77990_PD_3DG_A>;
+};
-- 
2.17.1

